website/src/controllers/api-v3/unsubscription.js

```diff
@@ -0,0 +1,57 @@
+import { model as User } from '../../models/user';
+import { model as EmailUnsubscription } from '../../models/emailUnsubscription';
+import { decrypt } from '../../libs/api-v3/encryption';
+import {
+  NotFound,
+} from '../../libs/api-v3/errors';
+
+let api = {};
+
+/**
+ * @api {post} /unsubscribe Unsubscribe an email or user from email notifications
+ * @apiVersion 3.0.0
+ * @apiName UnsubscribeEmail
+ * @apiGroup Unsubscribe
+ *
+ * @apiParam {String} code An unsubscription code
+ *
+ * @apiSuccess {String} okRes An message stating the user/email unsubscribed successfully
+ */
+api.unsubscribe = {
+  method: 'GET',
+  url: '/unsubscribe',
+  middlewares: [],
+  async handler (req, res) {
+    req.checkQuery({
+      code: {
+        notEmpty: {errorMessage: res.t('missingUnsubscriptionCode')},
```

Comment:
It looks like the new auth.js controller does this in api.registerLocal. I would like to pass the translations, because the invalid parameters error seems to vague. Although, I can't get this to return a new message :P